providers/intercom/metadata/schemas.json

```diff
@@ -328,6 +339,30 @@
         "updated_at": "updated_at",
         "workspace_id": "workspace_id"
       }
+    },
+    "tickets": {
+      "displayName": "Ticket List",
```

Comment:
This can be `Tickets` since this seems to be the only way to get all tickets from the API anyway

Reply:
Good catch. I will add a mapping to openapi script